### PR TITLE
Modify user page and message layout

### PR DESCRIPTION
**Pull request by anqitu**

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -89,7 +89,9 @@ public void storeMessage(Message message) {
         long timestamp = (long) entity.getProperty("timestamp");
         String text = (String) entity.getProperty("text");
         String imageUrl = (String) entity.getProperty("imageUrl");
-        String imageLabels = (String) entity.getProperty("imageLabels");
+        
+        @SuppressWarnings("unchecked")
```
Comment (amad-person) [inline src/main/java/com/google/codeu/data/Datastore.java]: Is there a better way to cast `List<Object>` as `List<String>`?

Check out:
1. https://stackoverflow.com/questions/1917844/how-to-cast-listobject-to-listmyclass/1920865
2. https://stackoverflow.com/questions/933447/how-do-you-cast-a-list-of-supertypes-to-a-list-of-subtypes

File: src/main/java/com/google/codeu/data/Message.java [hunk 2]
```diff
@@ -26,7 +26,8 @@
   private long timestamp;
   private String text;
   private String imageUrl;
-  private String imageLabels;
+  // private String imageLabels;
```
Comment (amad-person) [inline src/main/java/com/google/codeu/data/Message.java]: Is this commented out for future reference?

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 3]
```diff
@@ -104,9 +104,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
       message.setImageUrl(imageUrl);
       
-      String imageLabelsStr = imageLabels.stream()
-            .map(label -> label.getDescription()).collect(Collectors.joining(", "));
-      message.setImageLabels(imageLabelsStr);
+      // String imageLabelsStr = imageLabels.stream()
+      //       .map(label -> label.getDescription()).collect(Collectors.joining(", "));
+      // message.setImageLabels(imageLabelsStr);
```
Comment (amad-person) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: Same as above

File: src/main/webapp/css/user-page.css [hunk 4]
```diff
@@ -2,30 +2,83 @@ body > .page-header {
...
+  height: 90vh;
+}
+
+#user-profile-image {
+  object-fit: cover;
+  width: 290px;
+  height: 290px;
+}
+
+profile-container {
```
Comment (amad-person) [inline src/main/webapp/css/user-page.css]: This CSS class needs a `.` in front of it to be applied correctly I think

File: src/main/webapp/user-page.html [hunk 5]
```diff
@@ -60,32 +50,58 @@ <h1 class="display-4" id="page-title">My Page</h1>
           <input type="submit" value="Submit" />
         </form>
       </div>
-    </div>
+    </div> -->
```
Comment (amad-person) [inline src/main/webapp/user-page.html:53]: Need this code for future reference?

**Review by amad-person — CHANGES_REQUESTED** [review]
Just some questions, otherwise looks good

File: src/main/java/com/google/codeu/data/Message.java (hunk 2 above)
Comment (anqitu) [reply src/main/java/com/google/codeu/data/Message.java]: It was for the previous stage of development. Will remove!

File: src/main/webapp/css/user-page.css (hunk 4 above)
Comment (anqitu) [reply src/main/webapp/css/user-page.css]: Oh yes!

File: src/main/webapp/user-page.html (hunk 5 above)
Comment (anqitu) [reply src/main/webapp/user-page.html:53]: Yes. This is to fill in the about-me or any other relevant profile information.

Comment (amad-person) [reply src/main/webapp/user-page.html:53]: Alright! Thanks for clearing it up 😄

**Comment by amad-person** [comment]
Also, just to confirm: only the UI implementation for the search bar has been done right?

**Comment by anqitu** [comment]
> Also, just to confirm: only the UI implementation for the search bar has been done right?

Yup!

**Comment by anqitu** [comment]
Ok I have made all the changes accordingly.

**Review by amad-person — APPROVED** [review]

**Pull request merged**